src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/UpdateContestedCaseController.java

```diff
@@ -123,11 +123,8 @@ private void updateContestedFastTrackProcedureDetail(Map<String, Object> caseDat
     }
 
     private void updateContestedComplexityDetails(Map<String, Object> caseData) {
```

Comment:
This is now a method containing one method. How about moving the code from 
updateComplexityDetails into here and then deleting that method.